contents/newsletter/building-ai-features.md

```diff
@@ -0,0 +1,214 @@
+---
+title: The ultimate guide to building AI-powered features
+date: 2025-08-11
+author:
+ - ian-vanagas
+tags:
+ - feature flags
+---
+
+AI feels like a gold rush. Everyone’s staking claims and panning for quick wins. Few are building mines that create long-term value. Shiny demos are everywhere, but many are fool’s gold: bolted-on, rarely used, and quick to tarnish.
+
+We’ve all experienced poorly built AI-powered features. These can have a huge negative impact on user experience, hurting a company’s reputation and slowing adoption of future features.
+
+How do you avoid this fate? Choose the right AI-powered features to build, build them in the right way, and then continually work to improve them. This guide aims to help you do all three of these.
```

Comment:
I kind of like the gold rush analogy, but this intro isn't doing much for me atm. It kind of feels like the headline restated in a slightly different way.

As a potential different direction, I feel like we should talk about our own journey with AI here – e.g. I remember that line in the Max AI vid about how two years ago we didn't think AI was useful at all, but now we're totally rethinking our product.

I think this would add context and give us more credibility as an authority on the topic because we're actually doing it.